shell/platform/darwin/ios/ios_external_texture_gl.h

```diff
@@ -25,6 +25,9 @@ class IOSExternalTextureGL final : public Texture {
   fml::CFRef<CVOpenGLESTextureCacheRef> cache_ref_;
   fml::CFRef<CVOpenGLESTextureRef> texture_ref_;
   fml::CFRef<CVPixelBufferRef> buffer_ref_;
+  OSType pixelFormat = 0;
```

Comment:
These variables don't adhere to the google c++ style guide, use `pixel_format_`.  Same goes for down below.

Reply:
done.